lib/constants.go

```diff
@@ -478,6 +478,22 @@ func GetEncoderMigrationHeightsList(forkHeights *ForkHeights) (
 	return migrationHeightsList
 }
 
+type ProtocolVersionType uint64
+
+const (
+	ProtocolVersion0 ProtocolVersionType = 0
+	ProtocolVersion1
+	ProtocolVersion2
```

Comment:
Make these explicit so it's clear to future readers
```suggestion
	ProtocolVersion1 ProtocolVersionType = 1
	ProtocolVersion2 ProtocolVersionType = 2
```